CONTRIBUTING.md

```diff
@@ -29,7 +34,18 @@ Follow these steps to build a new version of the plugin:
 	```
 	This will compile the sources and place them in the ```bin``` folder. Any compilation errors will be displayed in the console.
 
-### Test
+## Test
+
+### Environment setup
+
+Make sure you can build the plugin by following the steps above.
+
+To run Android tests, make sure you have ```sdk\tools``` and  ```sdk\platform-tools``` in your PATH.
```

Comment:
actually i forgot to remove this in the plugin env setup, since this stuff is exclusive to running tests and not for building